src/pages/settings/Payments/PaymentsPage.js

```diff
@@ -145,6 +180,19 @@ class PaymentsPage extends React.Component {
                         }}
                         onItemSelected={method => this.addPaymentMethodTypePressed(method)}
                     />
+                    <ConfirmModal
```

Comment:
Why is this here instead of the transfer page? I must be missing something :smile:

Reply:
As per the docs, we take the user back to the Payment page and show the confirmation modal for the transfer.

Reply:
Ah I just looked back at the mockups and see that. Looks good!